qlty-check/src/tool/php.rs

```diff
@@ -243,6 +247,7 @@ pub mod test {
             reroute_tools_root(&temp_path, pkg);
 
             let composer = Composer {
+                workspace_root: temp_path.path().to_path_buf(),
```

Comment:
What is confusing to me about this line is that it seems to be assigning `temp_path` to the `Composer`'s `workspace_root`. However, we have another variable `self.workspace_root` so I would expect we would be assigning that to it.

Reply:
Whoops, sorry I missed the `mod test`!